### DISCUSSION TITLE
Allow for time offset in Explore Compare Mode

### DISCUSSION THREAD

**Discussion by PeterZaitsev** [grafana/grafana#49285] [Explore]
Explore Mode has great feature allowing to duplicate the query and see query results side by side, for example for different time range.    There is also great feature to lock time ranges between queries so they match.   What is missing though is being able to look them with offset (1h, 1d 1w etc)  so you can compare same query over 2 different but aligned time ranges which is often very helpful.

For example might be offset option can be added  to time picker menu or might be somewhere else 

![image](https://user-images.githubusercontent.com/3651746/169408687-ede75c1c-a6ab-4dab-a484-88e0431e1027.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
